apps/api-wallet/src/app/auth/auth.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable, NotAcceptableException } from '@nestjs/common';
+import { WalletService } from '../wallet/wallet.service';
+import { JwtService } from '@nestjs/jwt';
+
+@Injectable()
+export class AuthService {
+  constructor(
+    private readonly walletService: WalletService,
+    private jwtService: JwtService
+  ) {}
+  async validateUser(username: string, password: string): Promise<any | null> {
+    const user = await this.walletService.validate(username, password);
+    if (!user) return null;
+    // const passwordValid = await bcrypt.compare(password, user.password);
```

Comment:
remove this lang

Reply:
tried removing bai pero nagcause mag error

Reply:
the commented code?

Reply:
done